### DISCUSSION TITLE
In stats, show US state click count

### DISCUSSION THREAD

**Discussion by jxjj** [UMN-LATIS/z#283] [Ideas]
Feature Request: https://tdx.umn.edu/TDNext/Apps/29/Tickets/TicketDet.aspx?TicketID=1761255

In Z-link stats it would be really nice to be able to identify by state, where clicks are coming from. Currently there's only a country-by-country breakdown.

Maybe adding a US state map below the country map?

**Comment by cmcfadden** [comment]
our currently GeoIP database doesn't provide that level of granularity, so we'd need to source a new one.
